sprite.go

```diff
@@ -61,7 +61,96 @@ const (
 	AnimChannelMove  string = "@move"
 )
 
-type Sprite struct {
+type Sprite interface {
```

Comment:
https://github.com/goplus/spx/issues/316#issuecomment-2362719724

Now var declaration like

```go
var (
	Enemy Sprite
)
```

will not trigger auto-binding, while we expect it to.

Reply:
done